docs/pages/database-access/guides/redis-cluster.mdx

```diff
@@ -0,0 +1,185 @@
+---
+title: Database Access with Redis Cluster
+description: How to configure Teleport Database Access with Redis Cluster.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="9.0"
+>
+  Database access for Redis is available starting from Teleport `9.0`.
+</Details>
+
+If you want to configure Redis Standalone guide is available [here](redis.mdx).
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Configure mutual TLS authentication between Teleport and Redis Cluster.
+- Connect to Redis through Teleport.
+
+## Prerequisites
+
+- Teleport version `9.0` or newer.
+- Redis version `6.0` or newer.
+- `redis-cli` installed and added to your system's `PATH` environment variable.
+
+<Admonition type="note" title="Note">
+  Redis `7.0` and RESP3 (REdis Serialization Protocol) are currently not supported.
+</Admonition>
+
+## Step 1/6. Set up Teleport Auth and Proxy
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+## Step 2/6. Create a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+## Step 3/6. Create Redis users
+
+(!docs/pages/includes/database-access/redis-create-users.mdx!)
+
+## Step 4/6. Set up mutual TLS
+
+(!docs/pages/includes/database-access/tctl-auth-sign.mdx!)
```

Comment:
I think there should be a clearer transition between the use of the `tctl-auth-sign.mdx` partial and the `### Create the secrets` section. Added a suggestion:

```suggestion
(!docs/pages/includes/database-access/tctl-auth-sign.mdx!)

We will show you how to use the `tctl auth sign` command below.
```